docs/ui/organizations.md

```diff
@@ -12,17 +12,85 @@ tags:
+
+### Inviting organization members
+
+To invite new members to an organization in Prefect Cloud, select the **+** icon. Provide an email address and  organization role. You may add multiple email addresses.
+
+![Invite new members to an organization in Prefect Cloud.](/img/ui/org-invite-members.png)
+
+The user will receive an invite email with a confirmation link at the address provided. If the user does not already have a Prefect Cloud account, they must create an account before accessing the organization.
+
+When the user accepts the invite, they become a member of your organization and are assigned the role from the invite. The member's role can be changed (or access removed entirely) by an organization Admin.
+
+The maximum number of organization members varies. See the [Prefect Cloud plans](https://www.prefect.io/pricing) to learn more about options for supporting more users, service accounts, and workspaces.
+
+## Service accounts
+
+Service accounts enable you to create a Prefect Cloud API key that is not associated with a user account. Service accounts are typically used to configure API access for running agents or executing flow runs on remote infrastructure.
+
+Select **Service Accounts** to view, create, or edit service accounts for your organization.
+
+![Viewing service accoutns for an organization in Prefect Cloud.](/img/ui/service-accounts.png)
+
+Service accounts are created at the organization level, but may be shared to individual workspaces within the organization. See [workspace sharing](#workspace-sharing) for more information.
+
+!!! tip "Service account credentials"
+    When you create a service account, Prefect Cloud creates a new API key for the account and provides the API configuration command for the execution environment. Save these to a safe location for future use. If the access credentials are lost or compromised, you should regenerate the credentials from the service account page.
+
+## Workspace sharing
+
+You may give organization members and service accounts access to workspaces within an organization. Each workspace within an organization may have its own members and service accounts with roles and permissions specific to that workspace. Organization Admins have full access to all workspaces in an organization.
+
+Within a workspace, select **Workspace Sharing**, then select the **+** icon to add new members or service accounts to the workspace. Only organization Admins and workspace Owners may add members or service accounts to a workspace.
+
+![Organization workspace sharing in Prefect Cloud.](/img/ui/org-workspace-sharing.png)
+
+Members and service accounts must already be configured for the organization. An Admin or Owner may configure a different role for the user or service account as needed.
+
+!!! note "Default workspace role"
```

Comment:
also probably worth noting the default role does not apply to service accounts, only users